testflight/fixtures/archive-pipeline-3.yml

```diff
@@ -0,0 +1,39 @@
+---
+resources:
+- name: some-resource
+  type: mock
+  source:
+    create_files:
+      pipeline.yml: |
+        ---
+        jobs:
+        - name: normal-job
+          public: true
+          plan:
+          - task: a-task
+            config:
+              platform: linux
+              image_resource:
+                type: mock
+                source: {mirror_self: true}
+              run:
+                path: echo
+                args: ["hello world"]
+      name.yml: |
+        ---
+        name: somebody
+
+jobs:
+- name: sp-1
+  public: true
+  plan:
+    - get: some-resource
+    - set_pipeline: ((pipeline_name))
+      team: ((team_name))
```

Comment:
Is there a reason the `team` is configurable here (sim. in `archive-pipeline-1.yml`)? Based on the discussion here https://github.com/concourse/concourse/pull/5729#issuecomment-641392557, I believe the intention is to eventually get rid of `set_pipeline.team`, and it's just meant to be a temporary solution until we figure out the "right" way.

Maybe worth getting rid of it so we won't need to change these tests if/when we do get rid of the field.